### PR TITLE
fix buildkite erlang version

### PR DESCRIPTION
**Pull request by PrajaktaPurohit**
Signed-off-by: Christopher A. Snapp <csnapp@chef.io>

### Description

Correct version of erlang for the unit-tests in the verify pipeline.

### Adhoc Build

https://buildkite.com/chef/chef-chef-server-master-omnibus-adhoc/builds/661

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: src/bookshelf/Makefile [hunk 1]
```diff
@@ -28,7 +28,7 @@ include Makefile.db
 
 ## TODO: put 'ct' back in here
 all: $(REBAR3) elvis
-	@$(REBAR3) do clean, compile, eunit, dialyzer, xref
+	@$(REBAR3) do clean, compile, eunit, xref #, dialyzer, xref
```
Comment (markan) [inline src/bookshelf/Makefile]: Why are we disabling dialyzer and warnings_as_errors below?

We should at least file an issue to fix whatever the problem is and turn dialyzer and warnings_as_errors back on

Comment (PrajaktaPurohit) [reply src/bookshelf/Makefile]: yes absolutely - this was to test if there are any other unit test failures other than dialyzer.

**Comment by PrajaktaPurohit** [comment]
closes #1782

File: src/chef-mover/_checkouts/oc_erchef/include/chef_types.hrl [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
 %% Copyright 2012 Opscode, Inc. All Rights Reserved.
```
Comment (PrajaktaPurohit) [inline src/chef-mover/_checkouts/oc_erchef/include/chef_types.hrl]: Update the copyright to 2019!

Comment (lbakerchef) [reply src/chef-mover/_checkouts/oc_erchef/include/chef_types.hrl]: Done.

**Comment by lbakerchef** [comment]
The correct branch is now:  lbaker/fix_erlang_build_version
https://buildkite.com/chef/chef-chef-server-master-verify/builds/820#c441a902-a648-439f-b532-f1b053ba4635
https://buildkite.com/chef/chef-chef-server-master-omnibus-adhoc/builds/693#494e0bd8-b1b2-4bf0-a6ce-00bf871c443d

File: src/bookshelf/Makefile (hunk 1 above)
Comment (lbakerchef) [reply src/bookshelf/Makefile]: @markan - warnings, dialyzer, linting et al. have now been re-enabled.  In the words of Obi-wan, "warnings and linting are our specialty." :)

**Comment by lbakerchef** [comment]
Closed in favor of: https://github.com/chef/chef-server/pull/1817

**Pull request closed without merging**